octobot_trading/personal_data/orders/orders_manager.py

```diff
@@ -93,6 +95,12 @@ def remove_order_instance(self, order):
                                 f"{order.symbol}: {order.origin_quantity} at {order.origin_price} "
                                 f"(id: {order.order_id})")
 
+    def update_order_id_key(self, previous_id, order):
```

Comment:
Why `id_key`? I don't understand this method name

Reply:
it's `key` as in a dict since orders are referenced by their `id` in orders_manager. I don't think we should call it `update_order_id` as it's not changing the `id` attribute of the order itseft. 
What about `replace_order` ?

Reply:
👍